docs/wallet-integration-guide/examples/scripts/exchange-integration/tap-party.ts

```diff
@@ -0,0 +1,38 @@
+import { WalletSDK } from '@canton-network/wallet-sdk'
+import { KeyPair } from '@canton-network/core-signing-lib'
+import { PartyId } from '@canton-network/core-types'
+import { v4 } from 'uuid'
+import { pino } from 'pino'
+
+export async function tapParty(
```

Comment:
```suggestion
export async function tapDevNetFaucet(
```

Reply:
"tapping a party" is confusing as it is sort of proper English but non-sensical.